src/components/ProductItem.jsx

```diff
@@ -46,6 +66,19 @@ const ProductItem = ({ book:
                 New Arrival
             </span>:
     null;
+
+    const handleAddToCart = async () => {
+        const productPostedToCart = await postToCart(book, token);
+    
+        if(productPostedToCart) {   
+            showToast('Book added to the cart', 'success');
```

Comment:
Can we put these Toast messages in constants and then use them

Reply:
We could if we had the same messages used for all the cases. But since the messages are different in every case, I thought it did not make sense to create constants. Maybe for the themes, it makes sense.